test/cmocka/src/audio/selector/selector_test.c

```diff
@@ -0,0 +1,342 @@
+/*
```

Comment:
put the cmocka tests in separate patch (in the same PR), just to spit code and test?